components/Header.tsx

```diff
@@ -2,27 +2,86 @@ import Container from 'react-bootstrap/Container'
 import Nav from 'react-bootstrap/Nav'
 import Navbar from 'react-bootstrap/Navbar'
 import { useTranslation } from 'next-i18next'
+
+import { HeaderIcon } from './HeaderIcon'
+
+import { UrlPrefixes } from '../types/common'
 import getUrl from '../utils/browser/getUrl'
 
 export interface HeaderProps {
   userArrivedFromUioMobile: boolean
+  assetPrefix: string
+  urlPrefixes?: UrlPrefixes
 }
 
-export const Header: React.FC<HeaderProps> = ({ userArrivedFromUioMobile = false }) => {
+export const Header: React.FC<HeaderProps> = ({ userArrivedFromUioMobile = false, urlPrefixes, assetPrefix = '' }) => {
   const { t } = useTranslation('common')
 
   // Return a link back to:
   //   UIO Mobile landing page if user arrived from UIO Mobile
   //   main UIO landing page if user arrived from main UIO
-  const uioHomeLink = userArrivedFromUioMobile ? getUrl('uio-mobile-home-url') : getUrl('uio-desktop-home-url')
+  const uioHomeDesktopLink = getUrl('uio-desktop-home-url', urlPrefixes)
+  const uioHomeLink = userArrivedFromUioMobile ? getUrl('uio-mobile-home-url', urlPrefixes) : uioHomeDesktopLink
+
+  let globalHeader: JSX.Element
+
+  /* eslint-disable no-constant-condition */
+  if (true) {
```

Comment:
Just confirming that we are releasing the _code_ for Global Header, but it is hardocded inactive here.